lib/iris/__init__.py

```diff
@@ -113,6 +113,7 @@ def callback(cube, field, filename):
 from iris._deprecation import IrisDeprecation, warn_deprecated
 import iris.fileformats
 import iris.io
+import iris.options
```

Comment:
`iris.options` is not being used in this module, so by importing it here, you are catering to someone being able to do:
```
import iris
iris.options.parallel(scheduler='multiprocessing', num_workers=6)
```
I don't think this is a good thing. I would prefer users have to import iris.options directly